language/tools/move-cli/src/main.rs

```diff
@@ -96,6 +96,9 @@ pub enum SandboxCommand {
         /// Set this flag to ignore breaking changes checks and publish anyway
         #[structopt(long = "ignore-breaking-changes")]
         ignore_breaking_changes: bool,
+        /// fix publishign order
```

Comment:
`compile` with no additional arguments compiles everything in the current directory, whereas specifying a list of files will compile only those files. I think it might make sense to file that same convention here, especially since `sandbox publish` with no other arguments already publishes everything. We could do this by using `modules: Vec<String>` as an anonymous argument instead of adding `override-ordering`.

Reply:
Yup, that would help us eliminate an additional argument. The only issue is that we could have a single Move file containing multiple modules, if we specify at the file granularity, we lose two capabilities
1. the capability of publishing a subset of the modules in the same file, 
2. the capability of publishing the modules at a different ordering.

These issue can be addressed with the `override-ordering` flag, but at the cost of one extra flag in the command line